packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -245,6 +245,13 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
                 if (token.isCancellationRequested) {
                     return;
                 }
+                // Sort the list of root folder nodes by URI.
+                Array.from(this.resultTree.values())
+                    .sort(((a: SearchInWorkspaceRootFolderNode, b: SearchInWorkspaceRootFolderNode) => this.compare(a.folderUri, b.folderUri)))
```

Comment:
I don't think this `.sort()` works as it is supposed to: It sorts the array created by line 249, not the data `this.resultTree` has.

Reply:
Thank you @elaihau, I've updated the code, please let me know if there's still an issue :)